test_helpers/index.go

```diff
@@ -12,10 +13,15 @@ import (
 	"github.com/DATA-DOG/godog/gherkin"
 	"github.com/Originate/exosphere/src/application"
 	"github.com/Originate/exosphere/src/docker/compose"
+	"github.com/Originate/exosphere/src/docker/composebuilder"
 	execplus "github.com/Originate/go-execplus"
+	dockerTypes "github.com/docker/docker/api/types"
+	"github.com/moby/moby/client"
 	"github.com/pkg/errors"
 )
 
+var dockerComposeProjectName string
```

Comment:
Does this need to be a global variable for this file? Could we pass appName into killTestContainers?